homeassistant/components/matrix/manifest.json

```diff
@@ -2,8 +2,7 @@
   "domain": "matrix",
   "name": "Matrix",
   "documentation": "https://www.home-assistant.io/integrations/matrix",
-  "requirements": ["matrix-client==0.4.0"],
+  "requirements": ["matrix-nio==0.19.0", "pillow==9.3.0"],
   "codeowners": ["@tinloaf"],
```

Comment:
Please add yourself as a code owner.

Reply:
Done

Reply:
In accordance with https://github.com/home-assistant/core/pull/84701, in which `tinloaf` removed himself as a code owner, I am now listed as the sole codeowner; I'm happy to take on this role if this is the correct action to take.